### DISCUSSION TITLE
X Axis Label rotation

### DISCUSSION THREAD

**Discussion by marcuus** [ghalex/vue3-charts#28] [General]
Is it possible to rotate the x-axis labels so that long labels like timestamps can be displayed? Mine are currently being overwritten.
![image](https://user-images.githubusercontent.com/2087360/156964270-2a60af3a-8b27-4cf0-9eaa-d7de1e126791.png)

**Comment by ghalex** [comment]
Hi @marcuus,

You can use `rotate` property for primary axies:

```js
 <Chart
      :size="{ width: 800, height: 400 }"
      :data="data"
      :axis="{ primary: { rotate: true }}"
      :margin="margin"
      :direction="direction"
    >
....
```

this is available only from version 1.1.31

Thanks,
Alexandru

**Reply by CharlesOkwuagwu** [reply]
Please can we specify the rotation direction on the axis?
